GrandCentralBoard/Widgets/Harvest/View/AreaBarChartView.swift

```diff
@@ -100,15 +103,27 @@ final class AreaBarChartView : UIView, ViewModelRendering {
 
     private func handleTransitionFromState(state: RenderingState<ViewModel>, toState: RenderingState<ViewModel>) {
         switch (state, toState) {
+        case (.Waiting, .Rendering(let viewModel)):
+            configureWithViewModel(viewModel)
+
+            UIView.animateWithDuration(1) {
+                self.informationSheet.alpha = 0
+            }
+
         case (_, .Rendering(let viewModel)):
             configureWithViewModel(viewModel)
```

Comment:
What if we transition from .Failed to .Rendering ? Is it possible (information sheet would still be visible I think)?